lib/include/kickcat/Mailbox.h

```diff
@@ -163,23 +168,29 @@ namespace kickcat::mailbox::response
     {
         friend class AbstractMessage;
     public:
+        /// @deprecated
         /// \param max_msgs Max messages allowed simultaneously in the processing queue
         Mailbox(AbstractESC* esc, SyncManagerConfig mbx_in, SyncManagerConfig mbx_out, uint16_t max_msgs = 1);
+
+        Mailbox(AbstractESC* esc, uint16_t max_allocated_ram_by_msg, uint16_t max_msgs = 1);
         ~Mailbox() = default;
 
         void enableCoE(CoE::Dictionary&& dictionary);
         CoE::Dictionary& getDictionary(){return dictionary_;}
 
+        bool configure_sm(); // Configure sync managers
         void receive(); // Try to receive a message from the ESC
         void process(); // Process a message in the to_process_ queue if any
         void send();    // Send a message in the to_send_ queue if any, keep it in the queue if the ESC is not ready yet
 
     private:
         void replyError(std::vector<uint8_t>&& raw_message, uint16_t code);
+        std::tuple<uint8_t, SyncManager> find_sm(uint16_t controlMode);
```

Comment:
Same style remark

Reply:
Moved it to abstract esc, kept snake case as abstractESC function are in snake case... not really consistent but...